programs/ssl/ssl_client2.c

```diff
@@ -39,12 +39,22 @@
 
 #if !defined(MBEDTLS_ENTROPY_C) || \
     !defined(MBEDTLS_SSL_TLS_C) || !defined(MBEDTLS_SSL_CLI_C) || \
-    !defined(MBEDTLS_NET_C) || !defined(MBEDTLS_CTR_DRBG_C)
+    !defined(MBEDTLS_NET_C)
```

Comment:
What does `Support HMAC_DRBG in the TLS test programs ` have to do with `Build with only montgomery curves`? Shouldn't this commit be in a different PR?